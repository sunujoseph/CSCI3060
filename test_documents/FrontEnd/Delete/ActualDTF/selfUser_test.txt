02 admin1 AA 000000000
