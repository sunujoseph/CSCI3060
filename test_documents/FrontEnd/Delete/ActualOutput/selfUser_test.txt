path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Delete\ActualDTF\selfUser_test.txt
Enter Username: Error: Invalid Input
Error: Invalid Input
