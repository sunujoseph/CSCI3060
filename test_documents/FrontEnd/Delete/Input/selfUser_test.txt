login
admin1
delete
admin1
logout
