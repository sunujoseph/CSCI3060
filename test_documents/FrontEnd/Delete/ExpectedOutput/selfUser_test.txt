Enter user name: 
Enter user name to be deleted: 
Cannot delete your own account.
